test/attr/attr_autoclosure.swift

```diff
@@ -292,3 +292,7 @@ func sr_11938_3(_ x: [@autoclosure String]) {} // expected-error {{'@autoclosure
 
 protocol SR_11938_P {}
 struct SR_11938_S : @autoclosure SR_11938_P {} // expected-error {{'@autoclosure' may only be used on parameters}}
+
+// SR-11939
+func sr_11939(_ closure: @autoclosure () -> String...) {} // expected-error {{'@autoclosure' must not be used on variadic parameters}}
+sr_11939("A") // No crash
```

Comment:
This should be added into `validation-tests/compiler_crashers_2_fixed`? It's just a question since this was a crasher. I'm just curious :))

Reply:
You’re right! I’ll move it to the right directory.

Reply:
Done